library/model/book.py

```diff
@@ -31,28 +32,11 @@ def __init__(self, title, authors, publisher, pub_date, genres, pages, isbn, typ
         self.genres = genres
         self.pages = pages
         self.isbn = isbn
-        self._book_type = type
+        self.book_type = type
```

Comment:
Thats a great idea; however, you could do even better and provide a smoother solution with subclasses

Reply:
Note how I wrote this feedback in a very polite way and did not say your solution is bad